ytypes/node.go

```diff
@@ -257,7 +278,7 @@ func retrieveNodeList(schema *yang.Entry, root interface{}, path, traversedPath
 			if err != nil {
 				return nil, status.Errorf(codes.Unknown, "failed to convert the field value to string, field %v: %v", fieldName, err)
 			}
-			if pathKey != keyAsString {
+			if !(args.handleWildcards && pathKey == "*") && pathKey != keyAsString {
```

Comment:
That's the change for multiple keys!

Reply:
Ah, thanks -- I apologise, I'd missed that this wasn't reverted.